department-of-veterans-affairs/caseflow
# Show ScheduleHearingColocatedTask on case timeline

Resolves #14362

### Description
- ensure `ScheduleHearingColocatedTask` is visible on timeline with instructions 
- update seed script to generate more tasks

### Testing Plan
1. Login as `BVATWARNER ` and go to Hearing management queue
2. Click on `Unassigned` tab and filter tasks by `Confirm schedule hearing` value
2. Click on any of the tasks to go to case details page and copy `Veteran ID`
3.  Click on any of the two actions: [`Cancel task`, `Hearing does need to be scheduled, create schedule hearing task`]
4. Search for case by copied `Veteran ID` in `/search` and see the `ScheduleHearingColocatedTask` under Case Timline

### User Facing Changes
 BEFORE|AFTER
 ---|---
Action: `Cancel Task`
<img width="799" alt="Screen Shot 2020-06-01 at 5 16 59 PM" src="https://user-images.githubusercontent.com/20735998/83455405-d7d79400-a42b-11ea-8165-d041bf2df590.png"> | <img width="776" alt="Screen Shot 2020-06-01 at 5 17 35 PM" src="https://user-images.githubusercontent.com/20735998/83455418-db6b1b00-a42b-11ea-86b5-76e7a6d8d1f4.png">
Action:  `Hearing does need to be scheduled, create schedule hearing task`
<img width="789" alt="Screen Shot 2020-06-01 at 5 36 22 PM" src="https://user-images.githubusercontent.com/20735998/83456830-6fd67d00-a42e-11ea-8c38-f9b00da5ed6b.png"> | <img width="780" alt="Screen Shot 2020-06-01 at 5 35 58 PM" src="https://user-images.githubusercontent.com/20735998/83456833-736a0400-a42e-11ea-8b9c-26f4fea492e6.png">
